tests/testthat/test-check_tbl_value_col_ascending.R

```diff
@@ -163,3 +163,22 @@ test_that("(#78) check_tbl_value_col_ascending will sort even if the data doesn'
   expect_equal(nrow(actual), 1)
   expect_equal(actual, expected, ignore_attr = TRUE)
 })
+
+
+test_that("check_tbl_value_col_ascending works when output type IDs differ by target", {
+  hub_path <- test_path("testdata/hub-diff-otid-per-task/")
+  tbl <- arrow::read_csv_arrow(
+    fs::path(hub_path, "model-output/2024-01-10-ILI-model.csv")
+  ) %>%
+    hubData::coerce_to_character()
+  file_path <- "ISI-NotOrdered/2024-01-10-ILI-model.csv"
+  file_meta <- parse_file_name(file_path)
+  expect_s3_class(
+    check_tbl_value_col_ascending(tbl, file_path, hub_path, file_meta$round_id),
+    c("check_success", "hub_check", "rlang_message", "message", "condition"),
+    exact = TRUE
```

Comment:
I appreciate checking the class instead of adding another snapshot test. Removing the extra classes that we don't control will make this test less flaky. 

```suggestion
    "check_success"
```